File: ShadowFrameworkLite20_OpenGL20/src/shadow/pipeline/openGL20/SFGL20Vertex3fArray.java
package shadow.pipeline.openGL20;

import shadow.math.SFVertex3f;
import shadow.pipeline.SFArrayElementException;

public class SFGL20Vertex3fArray extends SFGL20ListData<SFVertex3f>{

	@Override
	protected void assignValues(SFVertex3f writing, SFVertex3f reading)
			throws SFArrayElementException {
		writing.set3f(reading);
	}
	
	@Override
	protected SFVertex3f generateGenericElement() {
		return new SFVertex3f(0, 0, 0);
	}
	
}

File: ShadowFrameworkJS/SF20OpenGL20Graphics/src/shadow/pipeline/openGL20/SFGL20Vertex3fArray.js

function SFGL20Vertex3fArray(){
}

SFGL20Vertex3fArray.prototype = {

	assignValues:function(writing, reading){
		writing.set3f(reading);
	},

	generateGenericElement:function(){
		return ,new ,SFVertex3f(0, 0, 0);
	}

};